RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate TeamChannelsView to Typescript

## Proposed changes

Migrate TeamChannelsView to Typescript

## Issue(s)	


## How to test or reproduce

- Create a Team
- Then inside the team, click on `#`
- This is the view migrated, if it is a new team, the list should be empty.
- To add channels to the team, click on `edit` button and `Create New` or `Add Existing`
- After doing this, back to view migrated and new channels should appear in the list

## Screenshots

## Types of changes




## Checklist



## Further comments
